### PR TITLE
add net.sourceforge.Lifeograph

### PR DESCRIPTION
**Pull request by blhps**
Pull request for Lifeograph to be added to Flathub

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build net.sourceforge.Lifeograph

File: net.sourceforge.Lifeograph.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
+{
+    "app-id": "net.sourceforge.Lifeograph",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "lifeograph",
+    "finish-args": [ "--socket=x11", "--socket=wayland", "--filesystem=host" ],
+    "build-options" : {
```
Comment (bilelmoussaoui) [inline net.sourceforge.Lifeograph.json]: Those build options are already set by default.

File: net.sourceforge.Lifeograph.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{
+    "app-id": "net.sourceforge.Lifeograph",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "lifeograph",
+    "finish-args": [ "--socket=x11", "--socket=wayland", "--filesystem=host" ],
```
Comment (bilelmoussaoui) [inline net.sourceforge.Lifeograph.json]: Nitpick: one arg per line.

File: net.sourceforge.Lifeograph.json [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": ["/include", "/lib/*/include", "/lib/pkgconfig", "/share/pkgconfig", "/share/aclocal", "/man", "/share/man", "/share/info", "/share/gtk-doc", "*.la", "*.a" ],
```
Comment (bilelmoussaoui) [inline net.sourceforge.Lifeograph.json]: Same as finish-args.

**Comment by bilelmoussaoui** [comment]
The build fails because the appdata file lacks OARS tag
```
builddir/files/share/appdata/net.sourceforge.Lifeograph.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
```

File: net.sourceforge.Lifeograph.json (hunk 2 above)
Comment (blhps) [reply net.sourceforge.Lifeograph.json]: resolved in 06866a5dea7ad4fea7401422926a8a6ba70424db

File: net.sourceforge.Lifeograph.json (hunk 3 above)
Comment (blhps) [reply net.sourceforge.Lifeograph.json]: addressed in 06866a5dea7ad4fea7401422926a8a6ba70424db

File: net.sourceforge.Lifeograph.json (hunk 1 above)
Comment (blhps) [reply net.sourceforge.Lifeograph.json]: addressed in 06866a5dea7ad4fea7401422926a8a6ba70424db

**Comment by blhps** [comment]
All issues are addressed. Now it should be ready to be merged.

**Comment by bilelmoussaoui** [comment]
bot, build net.sourceforge.Lifeograph

**Comment by nedrichards** [comment]
@blhps just to check, you are part of the upstream development group, right?

**Comment by barthalion** [comment]
Seems so! Going to merge it.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/net.sourceforge.Lifeograph


Thanks!

**Pull request closed without merging**